include/os/freebsd/spl/sys/isa_defs.h

```diff
@@ -159,6 +159,16 @@ extern "C" {
 #endif
 #define	_SUNOS_VTOC_16
 
+#elif defined(__loongarch__)
```

Comment:
On Linux this line is spelled as `#elif defined(__loongarch__) && defined(__loongarch_lp64)`.  Should here be the same or `lp64` is the only option there now and forever?

Reply:
Thank you for your suggestion. I have updated the code to be consistent with the Linux version. 
It compiles successfully (the porting work is not yet complete, and it has not been tested for normal operation). 
Currently, Loongson has not released any high-performance devices using loongarch32 (compared to desktop or server hardware),
so 32-bit loongarch can actually be temporarily disregarded.

Reply:
Though the question now is whether `__loongarch_lp64` will be defined on FreeBSD. So far is is not. I can see only `__loongarch__`.  So it feels like we need OS side to stabilize first.

Reply:
ok